src/plugins/makeModifiers.js

```diff
@@ -23,7 +23,10 @@
  *   // => ['modifier']
  *
  */
-export const maker = (block, { modifier }, { delimiters }) =>
-  modifier && modifier.split(/\s+/).map((item) => `${block}${delimiters.modifier}${item}`)
+export const maker = (block, { modifier = '' }, { delimiters }) => {
+  const mod = modifier.trim()
+
+  return mod && mod.split(/\s+/).map(item => `${block}${delimiters.modifier}${item}`)
```

Comment:
@robinpokorny could you open a PR with identity function usage? :) if only it will pass all the tests!